src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -467,7 +467,22 @@ private async Task WriteExecutableChange(ExecutableViewModel executableViewModel
                     && project.GetEffectiveLaunchProfile() is LaunchProfile launchProfile
                     && launchProfile.LaunchUrl is string launchUrl)
                 {
-                    endpointString += $"/{launchUrl}";
+                    if (!launchUrl.Contains("://"))
```

Comment:
This is probably simpler and effective enough, but any particular reason to avoid creating a Uri and checking IsAbsoluteUri?

Reply:
Does Uri provides better safety compared to current code? Keeping it simple may be useful here.

Reply:
Most of this just stemmed from my concerns with the URL concatenation (since a relative URL that starts with a `/` isn't supposed to just be concatenated). And then once I went down that route I saw other places where there's existing methods that _could_ be used. But since you said below that launchUrl can't start with a `/`, that's less of a concern and so I agree that simple is good. If you're not concerned about any of the things I mentioned, then I'm good with this.